### PR TITLE
Add CVE-2021-39222 for GHSA-xhxq-f4vg-jw5g

### PR DESCRIPTION
**Pull request by shelbyc**
Add CVE-2021-39222 for GHSA-xhxq-f4vg-jw5g

### CODE REVIEW

File: 2021/39xxx/CVE-2021-39222.json [hunk 1]
```diff
@@ -1,18 +1,113 @@
 {
```
Comment (cve-team) [inline 2021/39xxx/CVE-2021-39222.json:1]: 'ttps://hackerone.com/reports/1135481' does not match '^(ftp|http)s?://\\S+$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by ktodd229** [comment]
Please review the necessary changes, fix, and resubmit.

File: 2021/39xxx/CVE-2021-39222.json (hunk 1 above)
Comment (shelbyc) [reply 2021/39xxx/CVE-2021-39222.json:1]: Corrected to start with https

**Review by shelbyc — COMMENTED** [review]
Corrected the typo (ttps instead of https in URL)

**Pull request merged**